jwst/regtest/test_schema_editor.py

```diff
@@ -1,118 +0,0 @@
-"""Tests for the schema editor
-
-Notes
------
-The tests do not have coverage. The original code was written without tests.
-The tests here are for later modifications.
-"""
-import os
-from pathlib import Path
-import yaml
-
-import pytest
-
-import stdatamodels.jwst.datamodels.schema_editor as se
-
-from jwst.regtest.regtestdata import text_diff
-from jwst.lib.file_utils import pushdir
-
-# Define data locations in artifactory
-KEYWORD_DB = 'datamodels/keyword_db'
-FIXED_SCHEMA = 'fixed'
-SCHEMA_TRUTH = 'truth/test_schema_editor'
```

Comment:
Once merged these artifactory directories could be removed. The copy of the keyword dictionary looks to be from 2020.